src/Sulu/Bundle/ContactBundle/Resources/js/containers/ContactAccountSelection/ContactAccountSelection.js

```diff
@@ -0,0 +1,158 @@
+// @flow
+import React, {Fragment} from 'react';
+import {observer} from 'mobx-react';
+import {action, computed, observable, toJS} from 'mobx';
+import equals from 'fast-deep-equal';
+import {MultiItemSelection} from 'sulu-admin-bundle/components';
+import {MultiListOverlay} from 'sulu-admin-bundle/containers';
+import {translate} from 'sulu-admin-bundle/utils';
+import ContactAccountSelectionStore from './stores/ContactAccountSelectionStore';
+import contactAccountSelectionStyles from './contactAccountSelection.scss';
+
+type Props = {|
+    disabled: boolean,
+    onChange: (value: Array<Object>) => void,
```

Comment:
`Object` is basically an alias for `any` now. do we have any existing types for contacts or accounts?

Reply:
No, we don't have any types at the moment... I know that it is currently an alias for `any`, but changing that now might take some time... And the annotation can at least be read as some kind of documentation now, so the developer knows what's at least expected here...